examples/24_gaussian_splats.py

```diff
@@ -67,6 +71,63 @@ def load_splat_file(splat_path: Path, center: bool = False) -> SplatFile:
     }
 
 
+def load_ply_file(ply_file_path: Path, center: bool = False) -> SplatFile:
+    plydata = PlyData.read(ply_file_path)
+    vert = plydata["vertex"]
+    sorted_indices = onp.argsort(
```

Comment:
just wondering why does this need to be sorted?

Reply:
no clue but it seems to run on average 0.5-1 second faster this way when I timed it

Reply:
<img width="247" alt="Screenshot 2024-07-08 at 12 36 08 PM" src="https://github.com/nerfstudio-project/viser/assets/93736325/ce6f5a73-4bee-4474-9ffe-237cc1b6a7c5">